### PR TITLE
add docker image with java 17

### LINKED ISSUE #926: Java 17 (LTS) support
I wish that MTA build tool would support Java 17 (LTS) and provide all necessary images for it.

### PR DESCRIPTION
**Pull request by hilmarf**
Before submitting a pull request, please ensure the following:

## Description

Helping to fix #926

### Checklist
- [x] Extended the README / documentation, if necessary

### CODE REVIEW

**Comment by wlsc** [comment]
cc @allaVolkov

**Comment by wlsc** [comment]
@bd82 hi. could you take a look at this PR please? I've run these changes locally and it looks promising

File: Dockerfile_mbtci_java17 [hunk 1]
```diff
@@ -0,0 +1,90 @@
+FROM sapmachine:17
+
+# Build time arguments
+ARG MTA_USER_HOME=/home/mta
+ARG MBT_VERSION=1.2.8
```
Comment (wlsc) [inline Dockerfile_mbtci_java17:5]: last version is now below:
```suggestion
ARG MBT_VERSION=1.2.16
```

File: Dockerfile_mbtci_java17 [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+ARG GO_VERSION=1.15.6
```
Comment (wlsc) [inline Dockerfile_mbtci_java17:6]: maybe it makes sense also to update Golang (at least to a highest bugfix version)
```suggestion
ARG GO_VERSION=1.15.15
```

**Comment by hilmarf** [comment]
Dear CodeOwners (@Volkov / @ShimiT),
are you still actively work on this repo?

**Comment by hilmarf** [comment]
haha... looks like finally the requested images are there: https://registry.hub.docker.com/u/devxci

**Comment by wlsc** [comment]
@hilmarf yes, finally! I've migrated recently CAP-based service to Java 17 successfully.

**Pull request closed without merging**